docs/faq.md

```diff
@@ -21,7 +21,7 @@ the documentation sources.
 
 ### How much does Engine cost?
 
-Docker Engine is 100% free. It is open source, so you can use it without paying.
+The open source Docker Engine is available for free. Commercial support is also available for the Docker Engine as a standalone product or as part of the Docker Datacenter subscription. Pricing details are available at [https://www.docker.com/pricing](www.docker.com/pricing)
```

Comment:
@bettyjunod no header is needed since the fact that its in an open-source repo for an open-source project means its open-source docs.  If these were proprietary docs then I'd ask them to moved to another repo (the exception being if we had an entire section dedicated to info/pointers for lots of products, but we don't have that kind of section).